specs/Modal.spec.js

```diff
@@ -233,11 +235,31 @@ describe('State', () => {
   it('remove class from document.body when no modals opened', () => {
     renderModal({ isOpen: true });
     renderModal({ isOpen: true });
+    
     expect(isBodyWithReactModalOpenClass()).toBeTruthy();
     unmountModal();
     expect(isBodyWithReactModalOpenClass()).toBeTruthy();
     unmountModal();
-    expect(!isBodyWithReactModalOpenClass()).toBeTruthy();
+    expect(isBodyWithReactModalOpenClass()).toBeFalsy();
+  });
+  
+  it('should not add classes to document.body for unopened modals', () => {
+    renderModal({ isOpen: true });
+    expect(isBodyWithReactModalOpenClass()).toBeTruthy();
+    renderModal({ isOpen: false, bodyOpenClassName: 'testBodyClass' });
+    expect(isBodyWithReactModalOpenClass('testBodyClass')).toBeFalsy();
```

Comment:
Expectation should be always `True`. 

`expect(!isBodyWithReactModalOpenClass('testBodyClass')).toBeTruthy();`